containers/podman-desktop
# feat: allow extensions to customize icons (#1899)

### What does this PR do?

This PR allow extensions to customize icons in the container list.

As it uses the `when` clause which has to be supported by #1898 I only added the minimum code to support the `in`/`not in` operator. All the rest will be worked in that issue.

### Screenshot/screencast of this PR

![image](https://github.com/containers/podman-desktop/assets/49404737/7aae5347-4f07-4854-ba11-1f629b5ccf22)

### What issues does this PR fix or reference?

This solves part of #1899 

### How to test this PR?

1. go to the container list and check that the image of the kind container is different

N.B: The image is ugly as i was not able to create a good one when converting svg to woff. I asked @mairin to help me. When she has some free time she will make it better. I left my ugly one as a placeholder.
